addon/templates/components/bootstrap-input.hbs

```diff
@@ -4,6 +4,32 @@
   <label for="{{inputId}}" class="control-label {{if hasColumns labelColumnCss ''}} {{if customLabelCss customLabelCss ''}} {{if srOnly 'sr-only' ''}}">{{label}}</label>
   {{#if hasColumns}}
     <div class="{{inputColumnCss}}">
+      {{#if checkboxType}}
```

Comment:
No need for conditional, always pass checked=checked. If it isn't null, input will spit out an error. If you want an additional check in the component, decorate the checked property and use `Ember.assert`.